lms/resources/_js_config/__init__.py

```diff
@@ -327,13 +330,16 @@ def _add_canvas_speedgrader_settings(self, **kwargs):
                 "h_username": self._h_user.username,
                 "lis_result_sourcedid": lis_result_sourcedid,
                 "lis_outcome_service_url": lis_outcome_service_url,
-                "learner_canvas_user_id": self._request.params["custom_canvas_user_id"],
+                # Canvas sends custom_canvas_user_id as in integer in LTI1.3 and as a string in LT1.1
+                "learner_canvas_user_id": str(custom_canvas_user_id)
```

Comment:
I'm tempted to move this (and the same thing that we did in marshmallow in the previous PR) to LTIParams.

We can return the right value there the same way we tweak the value of `roles`.

https://github.com/hypothesis/lms/blob/master/lms/models/lti_params.py#L98


It might be better to have a couple more conditionals there, close to the params themselves than scattered around.